designsafe/static/scripts/rapid/html/index.html

```diff
@@ -31,7 +31,7 @@
           <div class="input-group">
             <input type="text" class="form-control" ng-model="vm.filter_options.search_text" ng-change="vm.search()"></input>
             <span class="input-group-btn">
-              <button class="btn btn-default"> <i class="fa fa-trash"></i></button>
+              <button class="btn btn-default"> <i class="fa fa-trash" ng-click="vm.clear_filter_text()"></i></button>
```

Comment:
what about just using `clear_filters()`.  This is totally debatable, but I feel like clearing all search options might be better